app/src/interfaces.ts

```diff
@@ -0,0 +1,107 @@
+import {InferType as Schema} from "yup";
+import {CreateRecipeSchema, IdentifierSchema, ShowRecipeSchema, UpdateRecipeSchema} from "./schemas";
+
+/**
+ * Interfaces
+ * Types that are used in the application.
+ * They are built from Yup objects. See schemas.js.
+ *
+ */
+
+/**
+ * @see ShowRecipeSchema
```

Comment:
I don't really understand why you are using jsdoc and not just typescript as is?

```ts
interface ShowRecipeType {
  id: string;
  name: string;
  ingredients: IdentifierType[];
  author: AuthorType;
}
```

Reply:
and 
```ts
interfase UserRecipeForm extends CreateRecipeForm{
  id: string;
}
```
as an example with extends.